src/build_targets.rs

```diff
@@ -103,7 +103,7 @@ impl BuildTargets {
                 let impl_lib = if env == "msvc" {
                     targetdir.join(format!("{}.dll.lib", lib_name))
                 } else {
-                    targetdir.join(format!("{}.dll.a", lib_name))
+                    targetdir.join(format!("lib{}.dll.a", lib_name))
```

Comment:
Also prefix depending on msvc or not on line 102 above?

Reply:
What do you?
If env==msvc import library has an output name `${libname}.dll.lib`, else (==gnu) it has `lib${libname}.dll.a`

Reply:
Not only `imp_lib` here, but also `shared_lib` on line 102 above